09-coap/test_spec09.py

```diff
@@ -28,42 +29,8 @@
 pytestmark = pytest.mark.rc_only()
 
 
-# pylint: disable=R0903
-class CordEPRegisterParser(ShellInteractionParser):
-    # TODO: provide upstream
-    def __init__(self):
-        self.comps = {
-            "rdaddr": re.compile(r"RD address:\s+"
-                                 r"(?P<rdaddr>coaps?://"
-
-                                 r"\[[0-9a-f:]+(%\S+)?\]:\d+)"),
-            "epname": re.compile(r"ep name:\s+(?P<epname>.+)$"),
-            "ltime": re.compile(r"lifetime:\s+(?P<ltime>\d+)s$"),
-            "regif": re.compile(r"reg if:\s+(?P<regif>\S+)$"),
-            "location": re.compile(r"location:\s+(?P<location>\S+)$"),
-        }
-
-    def parse(self, cmd_output):
-        res = {}
-        for line in cmd_output.splitlines():
-            for key, comp in self.comps.items():
-                m = comp.search(line)
-                if m is not None:
-                    try:
-                        res[key] = int(m.group(key))
-                    except ValueError:
-                        res[key] = m.group(key)
-        return res
-
-
-class Shell(Ifconfig):
-    # TODO: provide cord_ep and coap? ShellInteraction upstream
-    def cord_ep_register(self, addr, port=None, timeout=-1, async_=False):
-        cmd = "cord_ep register [{addr}]".format(addr=addr)
-        if port is not None:
-            cmd += ":{port:d}".format(port=port)
-        return self.cmd(cmd, timeout=timeout, async_=async_)
-
+class Shell(Ifconfig, CordEp):
+    # TODO: provide coap? ShellInteraction upstream
```

Comment:
```suggestion
    pass
```

As the `coap` commands are specific to the `examples/gcoap` application, it does not make sense to provide it in the centralized `riotcrtl_shell` module.